README.md

```diff
@@ -10,11 +10,95 @@ save your Fiddle either as a GitHub Gist or to a local folder. Once published
 on GitHub, anyone can quickly try your Fiddle out by just entering it in the
 address bar.
 
-<h3 align="center">
-  <a href="https://github.com/electron/fiddle/releases/latest">
-  Download Electron Fiddle for macOS, Windows, and Linux
-  </a>
-</h3>
+# Download
+
+<table class="is-fullwidth">
+</thead>
+<tbody>
+</tbody>
+  <tr>
+    <td>
+      <img src="./.github/images/windows.png" width="24"><br />
+      Windows
+    </td>
+    <td>
+      <span>32-bit</span>
+      <a href="https://github.com/electron/fiddle/releases/download/v0.19.0/electron-fiddle-0.19.0-win32-ia32-setup.exe">
+        💿 Installer
+      </a>
+      <br />
+      <span>64-bit</span>
+      <a href="https://github.com/electron/fiddle/releases/download/v0.19.0/electron-fiddle-0.19.0-win32-x64-setup.exe">
+        💿 Installer
+      </a>
+      <br />
+      <span>ARM64</span>
+      <a href="https://github.com/electron/fiddle/releases/download/v0.19.0/electron-fiddle-0.19.0-win32-arm64-setup.exe">
+        💿 Installer
+      </a>
+      <span>
+        ❓ Don't know what kind of chip you have? Hit start, enter "processor" for info.
+      </span>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <img src="./.github/images/macos.png" width="24"><br />
+      macOS
+    </td>
+    <td>
+      <span>Intel Processor</span>
+      <a href="https://github.com/electron/fiddle/releases/download/v0.19.0/Electron.Fiddle-darwin-arm64-0.19.0.zip">
+        📦 Standalone Zip
+      </a><br />
+      <span>Apple M1 Processor</span>
```

Comment:
nit: Based on apple's naming conventions, I'm guessing `M1` will be outdated by next year. 😅 

```suggestion
      <span>Apple Silicon Processor</span>
```